packages/driver/src/cy/commands/waiting.js

```diff
@@ -100,11 +100,16 @@ module.exports = (Commands, Cypress, cy, state) => {
       _.keys(cy.state('aliases')).includes(str.slice(1))) {
         specifier = null
       } else {
-        // potentially request, response or index
+        // potentially request, response
```

Comment:
`index` part is added at #4248. As far as I researched, there was no test to use a number with `cy.wait`. It's for `cy.get`.

Maybe I might be wrong.